libs/media-player/src/lib/audio-annotator.tsx

```diff
@@ -0,0 +1,274 @@
+import { isNullOrUndefined } from '@coscrad/validation-constraints';
+import {
+    ArrowLeft as ArrowLeftIcon,
+    ArrowRight as ArrowRightIcon,
+    Clear as ClearIcon,
+} from '@mui/icons-material/';
+import { Box, IconButton, Stack, Typography, styled } from '@mui/material';
+import { useRef, useState } from 'react';
+import { FormattedMediaTime } from './formatted-currenttime';
+import { KeyboardKey, useKeyDown } from './use-key-down';
+
+/**
+ * This is a duplicate of the enum in the Audio component, it should live in
+ * it's own file
+ */
+export enum AudioMIMEType {
+    mp3 = 'audio/mpeg',
+    audioMp4 = 'audio/mp4',
+    audioOgg = 'audio/ogg',
+    // TODO change this to audio/wav. This requires a migration
```

Comment:
We can do this now. Notify Justin and Blake on slack. It doesn't require a migration any more.